tests/unit-tests/test_decoder.py

```diff
@@ -983,6 +986,385 @@ def test_MLPEFeatEdgeDecoder(h_dim, feat_dim, out_dim, num_ffn_layers):
         pred = out.argmax(dim=1)
         assert_almost_equal(prediction.cpu().numpy(), pred.cpu().numpy())
 
+@pytest.mark.parametrize("in_units", [16, 64])
+@pytest.mark.parametrize("num_classes", [4, 8])
+def test_DenseBiDecoder(in_units, num_classes):
+
+    u = th.tensor([0, 0])
```

Comment:
It looks like you use this same test graph data across many test functions. May create it in a separated function.